genie/input_to_database.py

```diff
@@ -166,21 +166,16 @@ def _check_valid(syn, filepaths, center, filetype, filenames,
     Function to validate a file
     '''
     # If no filetype set, means the file was named incorrectly
-    try:
-        valid, message, filetype = validate.validate_single_file(
-            syn,
-            filepaths,
-            center,
-            oncotreelink=oncotree_link,
-            testing=testing)
-        logger.info("VALIDATION COMPLETE")
-    except ValueError as e:
-        # Specify this as None for the single case where filename
-        # validation fails
-        filetype = None
-        message = str(e)
-        logger.error(message)
-        valid = False
+    valid, message, filetype = validate.validate_single_file(
+        syn,
+        filepaths,
+        center,
+        oncotreelink=oncotree_link,
+        testing=testing)
+    logger.info("VALIDATION COMPLETE")
+
+    valid, message = determine_validity_and_log(total_error, warning)
```

Comment:
This is unnecessary as it is already done in `validate_single_file`

Reply:
removed function as it isn't doing anything extra for validate single file